plugins/kvscheduler/refresh.go

```diff
@@ -97,8 +97,15 @@ func (scheduler *Scheduler) refreshGraph(graphW graph.RWAccess, keys utils.KeySe
 			if len(dump) == 1 {
 				plural = ""
 			}
-			scheduler.Log.Debugf("Descriptor %s dumped %d value%s: %v",
-				descriptor.Name, len(dump), plural, dump)
+
+			var dumpList string
+			for _, d := range dump {
+				dumpList += fmt.Sprintf("\n - %+v", d)
```

Comment:
hmm, I didn't include the new line because the logger ignores the character and puts everything into one line - at least that is the case in Contiv - or can the behavior be changed in the logger configuration?

Reply:
Yes, the logger output can be changed via formatters. 

And this way the formatter used for console output is properly formatted to make it easier to read during development while it doesn't alter the output that is currently printed for non-console (pipe to file) very much beside replacing newlines with `\n` character.